gitcoinco/web
# Update links for the Idena web client

##### Description

1. Connect Idena<>Gitcoin using Idena web client ([https://app.idena.io](https://app.idena.io)) 
2. Add links to how-to instructions ([https://idena.io/gitcoin](https://idena.io/gitcoin))
3. Update Idena logo

##### Testing

1. Click Connect button at the Trust Bonus tab to open Idena web app;
2. Create a new Idena account, skip entering the invitation code when requested;
3. Authorise Gitcoin to access your new Idena account;
4. Go to the Trust  Bonus tab: a newly created Idena account should be linked to Gitcoin.
